routes/foodtrucks.go

```diff
@@ -149,3 +167,117 @@ func GetFoodTrucksHandler(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	json.NewEncoder(w).Encode(foodTrucks)
 }
+
+func PutFoodTrucksHandler(w http.ResponseWriter, r *http.Request) {
+
+	// Checks for food truck ID
+	params := mux.Vars(r)
+	foodTruckID, foodTruckIDExists := params["foodTruckID"]
+	if !foodTruckIDExists {
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	// Get user from context
+	user, userLoggedIn := r.Context().Value(middleware.UserKey).(string)
+
+	fmt.Println(user)
+
+	// Check for a user, or if the user agent is from the scraper
+	if !userLoggedIn && r.Header.Get("User-Agent") != "MunchCritic/1.0" {
```

Comment:
Do we want the scraper to be able to edit food trucks?

Reply:
I think there's a problem/thing we have to think about of how an owner can create a food truck but what if that food truck also comes from Yelp or Places API

Reply:
hmm, the owner should be able to claim it, not sure if we want to add that functionality tho